### PR TITLE
Heading change

### PR DESCRIPTION
**Pull request by raghavmathur1**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,121 +1,198 @@
...
-    <ul class="grid">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="index.css" />
+    <title>Mountain Tranvel</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Open+Sans"
+      rel="stylesheet"
```
Comment (ArnabBiswas2303) [inline index.html:11]: This looks awful! Please remove the meta tags

something like
`<meta> This is good </meta>`

File: index.html [hunk 2]
```diff
@@ -1,121 +1,198 @@
...
+    />
+  </head>
+  <body>
+    <!-- THE HEADER -->
+    <header>
+      <h2><a href="#">Mountain Travel By Dcoders</a></h2>
+      <nav>
+        <li><a href="#">Tours</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contacts</a></li>
```
Comment (ArnabBiswas2303) [inline index.html]: Please do the following:
- [ ]  Change all to small case
- [ ]  Remove anchor tag

**Review by ArnabBiswas2303 — CHANGES_REQUESTED** [review]
Please make the changes accordingly

File: index.html (hunk 1 above)
Comment (raghavmathur1) [reply index.html:11]: This isn't possible

File: index.html (hunk 2 above)
Comment (raghavmathur1) [reply index.html]: Changes Done

Comment (ArnabBiswas2303) [reply index.html]: Awesome work! Thanks 💣

**Pull request merged**